media/js/src/wizard.js

```diff
@@ -5,11 +5,61 @@
         this.sectionSelector = sectionSelector;
         this.sections = $(this.sectionSelector);
         this.current = 0;
+        this.questionsAnswered = [
+           // 1 - 6 are houghton
+           false, false, false, false, false, false,
+           // 7 - 22 are abcs. the values don't matter
+           false, false, false, false, false, false,
+           false, false, false, false, false, false,
+           false, false, false, false,
+           // balance 1
+           false,
+           // balance 2
+           false
+        ];
+        this.registerRadioListeners();
         this.hideAllSections();
         this.addNavButtons();
         $(this.sections[this.current]).show();
     };
 
+    Wizard.prototype.allHoughtonQuestionsAnswered = function() {
+        for (var i = 0; i < 6; i++) {
+            if (!this.questionsAnswered[i]) {
+                return false;
+            }
+        }
+        return true;
+    };
+
+    Wizard.prototype.registerRadioListeners = function() {
+        var self = this;
+
+        var registerRadioListener = function(i) {
+            $('input[name="q' + i + '"]').change(function(e) {
+                self.questionsAnswered[i - 1] = $(this).is(':checked');
+                if (self.allHoughtonQuestionsAnswered()) {
+                    $('#next-button-0').removeAttr('disabled');
+                }
+                if (self.questionsAnswered[22]) {
+                    $('#next-button-2').removeAttr('disabled');
+                }
+                if (self.questionsAnswered[23]) {
+                    $('#next-button-3').removeAttr('disabled');
+                }
+            });
+        };
+
+        // register houghton listeners
+        for (var i = 1; i <= 6; i++) {
+            registerRadioListener(i);
+        }
+        // balance 1
+        registerRadioListener(23);
+        // balance 2
+        registerRadioListener(24);
```

Comment:
The hard-coded array indices here are a bit concerning. Another way to approach this is might be to select all the inputs on the selected wizard page, validate them, then enable the page's next button. An example of bulk clientside form validation is here:  https://github.com/ccnmtl/worth2/blob/master/media/js/src/views/quiz-validator.js